ush/config_defaults.sh

```diff
@@ -2096,6 +2103,21 @@ DO_NLDN_LGHT="FALSE"
 RADARREFL_MINS=(0 1 2 3)
 RADARREFL_TIMELEVEL=(0)
 
+#
+#-----------------------------------------------------------------------
+#
+# Parameters for Python ensemble postprocessing.
+#   time level (minute) from the hour that the NSSL mosaic files will be generated 
+# FFG_LATEST:
+#   location of latest flash flood guidance grid on /public
+# WGRIB2_IPOLATES:
+#   location of wgrib2 executable that has been set up with the IPOLATES library turned on
+#
+#-----------------------------------------------------------------------
+#
+FFG_LATEST=""
```

Comment:
I think config_defaults.sh already has the location of realtime FFG files under FFG_DIR.

Reply:
Cool!  It looks fine to me otherwise, although probably good for Ming or Guoqing to check.